Generator/Sources/generate/main.swift

```diff
@@ -8,6 +8,7 @@ ProcessInfo.applicationIdentifier = "ca.lacc.Website"
 let site = Site<Localization>(
     repositoryStructure: repositoryStructure,
     domain: domain,
+    localizationDirectories: localizatonDirectory,
```

Comment:
This localization stuff isn’t actually used by your LACC site (because it’s only in one language), but there needed to be something to pass to the function parameter.